File: javascript-solutions/fizzbuzz.js
var fs  = require("fs");

fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line !== "") {       
        var arr = line.split(" "); // Splits the string by space
        var output = [];
        for (i=1;i<arr[2]+1;i++) {
        	if (i%(parseInt(arr[0])) === 0 && i%(parseInt(arr[1])) === 0) {
        		output.push("FB");
        	} else if (i%(parseInt(arr[0])) === 0) {
        		output.push("F");
        	} else if (i%(parseInt(arr[1])) === 0) {
        		output.push("B");
        	} else {
        		output.push(i);
        	}
        }
    console.log(output.join(" "));  
    }
});

File: ruby-solutions/fizzbuzz.rb
File.open(ARGV[0]).each_line do |line|
  nums = line.split(" ")
	output = []
	nums.each do |num|
		if num.to_i%15 == 0
			output << "FB"
		elsif num.to_i%3 == 0
			output << "F"
		elsif num.to_i%5 == 0
			output << "B"
		else
			output << num
		end
	end
	p output.join(" ")
end